src/app.js

```diff
@@ -46,7 +46,14 @@ console.ignoredYellowBox = IGNORED_WARNINGS
 global.OS = Platform.OS
 global.slowlogOptions = { threshold: 16 }
 // Disable the font scaling
+if (!Text.defaultProps) {
```

Comment:
why is this needed?

Reply:
New RN does not initialize default props so if you try to access it without initializing it, you’ll throw.